<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFlightPlanComponent</class>
 <widget class="QTabWidget" name="CFlightPlanComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tb_FlightPlanMainTab">
   <attribute name="title">
    <string>Flight plan</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_MainTab">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gl_FlightPlan">
      <item row="11" column="2">
       <widget class="QPushButton" name="pb_Load">
        <property name="text">
         <string>Load plan</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <layout class="QGridLayout" name="gl_Remarks">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_Remarks">
          <property name="text">
           <string>11. Remarks</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="vs_Remarks">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pb_RemarksGenerator">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: transparent;
	border: none;
	text-align: left;
	text-decoration: underline;
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>Generator</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="pb_Reset">
        <property name="text">
         <string>Reset plan</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="le_EstimatedTimeEnroute">
        <property name="inputMask">
         <string>99:99</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="le_CruiseTrueAirspeed">
        <property name="text">
         <string>0 kts</string>
        </property>
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbl_Aircraft_Type">
        <property name="text">
         <string>3. Aircraft type</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QLabel" name="lbl_Route">
        <property name="text">
         <string>8. Route</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="lbl_FuelOnBorad">
        <property name="text">
         <string>12. Fuel on board</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_OriginAirport">
        <property name="inputMask">
         <string>&gt;AAAA</string>
        </property>
        <property name="text">
         <string>ICAO</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_AircraftType"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="le_TakeOffTimePlanned">
        <property name="inputMask">
         <string>99:99</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="le_AlternateAirport">
        <property name="inputMask">
         <string>&gt;AAAA</string>
        </property>
        <property name="text">
         <string>ICAO</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="lbl_AlternateAirport">
        <property name="text">
         <string>13. Alternate airport</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QPlainTextEdit" name="pte_Remarks">
        <property name="tabStopWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="le_FuelOnBoard">
        <property name="inputMask">
         <string>99:99</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="lbl_CrusingAltitude">
        <property name="text">
         <string>7. Cruising altitude</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lbl_TakeOffTimePlanned">
        <property name="text">
         <string>6. Departure time</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QPlainTextEdit" name="pte_Route"/>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QGroupBox" name="gb_Type">
        <property name="title">
         <string>1. Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rb_TypeVfr">
           <property name="text">
            <string>VFR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_TypeIfr">
           <property name="text">
            <string>IFR</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="le_DestinationAirport">
        <property name="inputMask">
         <string>&gt;AAAA</string>
        </property>
        <property name="text">
         <string>ICAO</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lbl_CruiseTrueAirspeed">
        <property name="text">
         <string>4. True airspeed</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QPushButton" name="pb_Send">
        <property name="text">
         <string>Send plan</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="le_CrusingAltitude">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_Callsign">
        <property name="text">
         <string>2. Callsign</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="le_PilotsName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="lbl_EstimatedTimeEnroute">
        <property name="text">
         <string>10. Estimated time enroute</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lbl_PilotsNameAndHomebase">
        <property name="text">
         <string>14. Pilots name / homebase</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_DestinationAirport">
        <property name="text">
         <string>9.Destination airport</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_OriginAirport">
        <property name="text">
         <string>5. Departure airport</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_Callsign">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>40</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLineEdit" name="le_PilotsHomebase"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lbl_LastSent">
        <property name="text">
         <string>Sent</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="3">
       <widget class="QLineEdit" name="le_LastSent">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="pb_ValidateFlightPlan">
        <property name="text">
         <string>Validate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_RemarksGenerator">
   <attribute name="title">
    <string>Remarks</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_RemarksGenerator">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gl_RemarksGenerator">
      <item row="12" column="2">
       <widget class="QFrame" name="fr_Buttons">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QPushButton" name="pb_Parse">
           <property name="text">
            <string>Parse</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pb_CopyOver">
           <property name="text">
            <string>Copy over</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lbl_RemarksGenerated">
        <property name="text">
         <string>Remarks</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_RequiredNavigationPerformance">
        <property name="text">
         <string>Required Navigation Performance</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QPlainTextEdit" name="pte_RemarksGenerated">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_AirlineOperator"/>
      </item>
      <item row="8" column="2">
       <widget class="QComboBox" name="cb_PilotRating">
        <item>
         <property name="text">
          <string>P0 - unrated</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_PerformanceCategory">
        <property name="text">
         <string>Performance category</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="cb_RequiredNavigationPerformance">
        <item>
         <property name="text">
          <string>VFR - not required</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 miles radius</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 miles radius</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_AircraftRegistration">
        <property name="text">
         <string>Aircraft registration</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_AirlineOperator">
        <property name="text">
         <string>Airline operator</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbl_PilotRating">
        <property name="text">
         <string>Pilot rating</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="BlackGui::CSelcalCodeSelector" name="frp_SelcalCode">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_NavigationEquipment">
        <property name="text">
         <string>Navigation equipment</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPlainTextEdit" name="pte_AdditionalRemarks"/>
      </item>
      <item row="7" column="2">
       <widget class="QComboBox" name="cb_VoiceCapabilities">
        <item>
         <property name="text">
          <string>Full voice</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Receive voice</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_AircraftRegistration"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lbl_Selcal">
        <property name="text">
         <string>SELCAL</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="cb_PerformanceCategory">
        <property name="toolTip">
         <string>Final Approach Speed</string>
        </property>
        <item>
         <property name="text">
          <string>VFR - not required</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A: below 91kts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B: 91-120kts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C: 121-140kts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D: 141-165kts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E: 166-210kts</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="cb_NavigationEquipment">
        <property name="toolTip">
         <string>How will you be navigating?</string>
        </property>
        <item>
         <property name="text">
          <string>VFR flying visually</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Direct VORs and NDBs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Default GPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GPS or FMC capable of SIDs/STARs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GPS oceantic certified</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lbl_AdditionalRemarks">
        <property name="text">
         <string>Additional remarks</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbl_VoiceCapabilities">
        <property name="text">
         <string>Voice capabilities</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="cb_NoSidsStarts">
        <property name="text">
         <string>no SIDs / STARs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_SidsStars">
        <property name="text">
         <string>SIDs / STARs</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CSelcalCodeSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/selcalcodeselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rb_TypeVfr</tabstop>
  <tabstop>rb_TypeIfr</tabstop>
  <tabstop>le_Callsign</tabstop>
  <tabstop>le_AircraftType</tabstop>
  <tabstop>le_CruiseTrueAirspeed</tabstop>
  <tabstop>le_OriginAirport</tabstop>
  <tabstop>le_TakeOffTimePlanned</tabstop>
  <tabstop>le_CrusingAltitude</tabstop>
  <tabstop>pte_Route</tabstop>
  <tabstop>le_DestinationAirport</tabstop>
  <tabstop>le_EstimatedTimeEnroute</tabstop>
  <tabstop>le_FuelOnBoard</tabstop>
  <tabstop>le_AlternateAirport</tabstop>
  <tabstop>pb_RemarksGenerator</tabstop>
  <tabstop>pte_Remarks</tabstop>
  <tabstop>le_PilotsName</tabstop>
  <tabstop>le_PilotsHomebase</tabstop>
  <tabstop>pb_Reset</tabstop>
  <tabstop>pb_Load</tabstop>
  <tabstop>pb_Send</tabstop>
  <tabstop>pb_Parse</tabstop>
  <tabstop>pb_CopyOver</tabstop>
  <tabstop>pte_RemarksGenerated</tabstop>
  <tabstop>le_AirlineOperator</tabstop>
  <tabstop>cb_PilotRating</tabstop>
  <tabstop>cb_RequiredNavigationPerformance</tabstop>
  <tabstop>pte_AdditionalRemarks</tabstop>
  <tabstop>cb_VoiceCapabilities</tabstop>
  <tabstop>le_AircraftRegistration</tabstop>
  <tabstop>cb_PerformanceCategory</tabstop>
  <tabstop>cb_NavigationEquipment</tabstop>
  <tabstop>cb_NoSidsStarts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
